proposals/0035-linalg-matrix.md

```diff
@@ -775,44 +794,54 @@ matrix object. When storing to `groupshared` memory, the matrix component data
 is converted to the target arithmetic or packed data type if the data types do
 not match.
 
-#### Matrix::Accumulate
+For the `Store` operations on `[RW]ByteAddressBuffers`, the `Stride` argument
+represents the row or column stride in bytes. For the `Store` operations on
+`groupshared` arrays, the `Stride` argument is the count of elements in the
+`groupshared` array.
+
+#### Matrix::InterlockedAccumulate
 
 ```c++
-template <MatrixScope ScopeLocal = Scope>
-typename hlsl::enable_if<Use == MatrixUse::Accumulator && ScopeLocal == Scope,
+
+// When Scope != Thread, the following overloads are available:
+template <MatrixUseEnum UseLocal = Use>
```

Comment:
I don't understand this template parameter `UseLocal`. It's not used except to make sure it's equal to `Use` (the default) which is required to be equal to `MatrixUse::Accumulator`. I see it on a number of methods where it's unused. Is this to satisfy some requirement elsewhere?